### ISSUE TITLE
Expose Metrics

### ISSUE THREAD

**Issue by slashmo** [slashmo/jaeger-client-swift#4]
`JaegerTracer` should expose metrics. Inspiration on which metrics to observe can be found in Jaeger's official Java client: https://github.com/jaegertracing/jaeger-client-java/blob/016e0c33be6b5327220b4048461527500842a077/jaeger-core/src/main/java/io/jaegertracing/internal/metrics/Metrics.java

**Comment by ktoso** [comment]
Yes that'll be excellent, you can view https://github.com/apple/swift-cluster-membership/commit/343e2ad672fbfc85c51c0043284261c2e34066ac how to nicely add metrics and have tests for them 👍
